schedule/yam/agama_extended.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 name: agama_extended
 description: >
-  Perform interactive installation with agama.
+  Perform interactive installation with agama setting a custom hostname.
```

Comment:
I think you interpretation of the ticket was different that what I thought but still a good idea to implement the same in both, UI and and unattended.
We could change this to:
"Perform interactive installation with the following additional test:
- Set static hostname via Agama profile